addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1271,11 +1287,13 @@ def _create_group(self, partners_to, default_display_mode=False, name=''):
             :rtype: dict
         """
         partners_to = OrderedSet(partners_to)
+        name = name or (', '.join(self.env['res.partner'].browse(partners_to).mapped('name')) + "..." if len(partners_to) > 5 else "")
```

Comment:
Can't we not given the name field as param at all in that case and let the compute do its job rather than repeating code?